facebook_downloader/downloader.py

```diff
@@ -28,6 +28,11 @@ def download_data():
                         config.app_secret(),
                         config.access_token())
     ad_accounts = _get_ad_accounts()
+    target_accounts = list(filter(None, config.target_accounts().split(',')))
+    if len(target_accounts) > 0:
+        logging.info('the app can see {} accounts but the configuration requested filtering for {} target accounts'.format(len(ad_accounts), len(target_accounts)))
```

Comment:
logging.info('... the configuration specified only %s target accounts: %s' , len(target_accounts), ', '.join(target_accounts))